src/styles/images/index.md.njk

```diff
@@ -8,53 +8,72 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Avoid unnecessary decoration. Only use images if there’s a real user need.
+Avoid using images for unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+This guidance is for government teams that build online services. To learn how to use images in GOV.UK mainstream content, go to [GOV.UK content guidance for images](https://www.gov.uk/guidance/content-design/images).
 
-### Photography
+## Using images to give information
 
-The aspect ratio for photographs should be 3:2.
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
 
+If you choose to add an image to your service, you’ll need to carefully decide if the image needs to be described with alternative text (alt text).
 
-### Illustrations or representative imagery
+### Alt text
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+Alternative text, or alt text, is read out by screen readers or displayed onscreen — usually when images do not load or have been disabled in browser settings.
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+All images should include the alt attribute, even if it’s left empty without alt text (`alt=""`).
```

Comment:
Might merge this into 1 paragraph, as it's 'getting in the way' of the main message for alt text.